drivers/sensor/tcs3472x/Kconfig

```
@@ -0,0 +1,67 @@
+# TCS3472x color sensor configuration options
+
+# Copyright (c) 2023, Thomas Kiss <thokis@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig TCS3472X
+	bool "TCS3472X sensor"
+	default y
+	depends on DT_HAS_AMS_TCS3472X_ENABLED
+	select I2C if $(dt_compat_on_bus,$(DT_COMPAT_AMS_TCS3457X),i2c)
```

You can simplify this line since the sensor only supports I2C (no SPI or other interfaces).
```suggestion
	select I2C
```